students/tri_nguyen/lesson02/fizz_buzz.py

```diff
@@ -0,0 +1,24 @@
+# -------------------------------------------#
+# Student:   Tri Nguyen
+# Instructor: Natasha Aleksandrova
+# Date:  02-Feb-2018
+# -------------------------------------------#
+
+# write a program that prints the numbers from 1 to 100
+# For multiple of three prints 'Fizz' instead of the numbers
+# For multiple of five print 'Buzz' instead of the numbers
+# For numbers which are multiple of both three and five print
+# 'FizzBuzz' instead
+
+# Declare variables
+num_range = range(1, 101)
+
+for number in num_range:
```

Comment:
more pythonic to use range inline with for loop since you're not using num_range again. like:
`for number in range(1, 101):`